src/pipeline/node/DetectionParser.cpp

```
@@ -24,6 +24,17 @@ namespace node {
 
 DetectionParser::~DetectionParser() = default;
 
+void DetectionParser::buildInternal() {
+    auto device = getDevice();
+    if(device) {
+        auto platform = device->getPlatform();
+        if(platform == Platform::RVC2 && properties.parser.decodeSegmentation) {
```

Cppcheck (2.20.0)

[information] Include file

(missingIncludeSystem)